tests/integration/security/reachability_test.go

```diff
@@ -147,6 +151,7 @@ func TestReachability(t *testing.T) {
 					expectCrossCluster: notFromNaked,
 					expectCrossNetwork: notNaked,
 					expectSuccess:      notToNaked,
+					minIstioVersion:    "1.15.0",
```

Comment:
does that mean it won't work for versions lesser than 1.15 ?

Reply:
1. If we try to test this master branch reachability tests against versions lesser than 1.15 then it won't work (so we are trying to skip) because of new API changes done(PR#39742) which are not there in version 1.13.
2. 1.13 branch already has its reachability tests which will work on 1.13 version.